src/graphql.rs

```diff
@@ -747,6 +748,13 @@ impl RepositoryMut {
         add: Vec<MarkersInput>,
         context: &Context,
     ) -> FieldResult<bool> {
+        {
+            let mut allow_refs = context.allow_refs.lock()?;
+            if !*allow_refs {
+                *allow_refs = true;
```

Comment:
It's kind of weird to me that this is done here and not up in the caller of parsed.execute (in the else). Wouldn't you risk forgetting to write it in some case down here?

Reply:
nevermind, this is the error reporting actually :)